chart/config/releases.yaml

```diff
@@ -48,6 +48,11 @@ releases:
   postgres:
     condition: features.autoscaler.enabled
     version: "39"
+  secret-rotation:
+    # secret-rotation is not a BOSH release.  It requires ruby & kubectl.
+    image:
+      repository: splatform/fissile-stemcell-sle
+      tag: SLE_15_SP1-27.4
```

Comment:
Not really happy with this; suggestions welcome.

Reply:
@mook-as create an issue for creating such a base (we'll likely have to do it on OBS)